data/json/bionics.json

```diff
@@ -92,7 +92,7 @@
     "type": "bionic",
     "name": "Battery System",
     "capacity": 10,
-    "description": "You have a battery draining attachment, and thus can make use of the energy contained in normal, everyday batteries.  Use 'E' to consume batteries.  Can be toggled on and off at will.",
+    "description": "You have a battery draining attachment, and thus can make use of the energy contained in normal, everyday batteries.  Use 'E' to consume battery charge.  Can be toggled on and off at will.",
```

Comment:
Maybe replace `consume` with absorb  or something that make clearer that the battery charges are transferred from a battery to your power storage.

Reply:
Yeah, that sounds better.